app/models/metric/targets.rb

```diff
@@ -68,6 +88,8 @@ def self.capture_vm_targets(targets, parent_class, options = {})
   def self.capture_targets(zone = nil, options = {})
     zone = MiqServer.my_server.zone(true) if zone.nil?
     zone = Zone.find(zone) if zone.kind_of?(Integer)
-    return capture_infra_targets(zone, options) + capture_cloud_targets(zone, options)
+    return capture_infra_targets(zone, options) + \
+      capture_cloud_targets(zone, options) + \
+      capture_container_targets(zone, options)
```

Comment:
@matthewd this seems a little smelly ... maybe you can shove this in the back of your brain for a later refactoring?